api/src/request-handlers/security/authentication.ts

```diff
@@ -0,0 +1,131 @@
+import { Request } from 'express';
```

Comment:
Nothing in this file is different, I just moved it into a new folder, and removed the old code that had to do with authorization (since it now lives in its own file).